src/Symfony/Installer/NewCommand.php

```diff
@@ -133,8 +134,8 @@ protected function checkSymfonyVersionIsInstallable()
             }
         }
 
-        // 2.0, 2.1, 2.2 and 2.4 cannot be installed because they are unmaintained
-        if (preg_match('/^2\.[0124]\.\d{1,2}$/', $this->version)) {
+        // 2.0, 2.1, 2.2, 2.4, 2.5 and 2.6 cannot be installed because they are unmaintained
```

Comment:
@javiereguiluz 

yes indeed

but reading http://symfony.com/roadmap?version=2.5#checker + http://symfony.com/roadmap?version=2.6#checker,
I thought it will be a good idea to only allow new dev to install maintained version :)